src/math/lp/lp_core_solver_base.h

```diff
@@ -45,136 +46,141 @@ X dot_product(const vector<T> & a, const vector<X> & b) {
     return r;
 }
 
-template <typename T, typename X> // X represents the type of the x variable and the bounds
-class lp_core_solver_base {    
+template <typename T, typename X>  // X represents the type of the x variable and the bounds
+class lp_core_solver_base {
     unsigned m_total_iterations;
     unsigned m_iters_with_no_cost_growing;
-    unsigned inc_total_iterations() { ++m_settings.stats().m_total_iterations; return m_total_iterations++; }
-private:
+    unsigned inc_total_iterations() {
+        ++m_settings.stats().m_total_iterations;
+        return m_total_iterations++;
+    }
+
+   private:
     lp_status m_status;
-public:
+
+   public:
     bool current_x_is_feasible() const {
-        TRACE("feas",
-              if (m_inf_heap.size()) {
-                  tout << "column " << *m_inf_heap.begin() << " is infeasible" << std::endl;
-                  print_column_info(*m_inf_heap.begin(), tout);
-              } else {
-                  tout << "x is feasible\n";
-              }
-              );
+        TRACE(
+            "feas",
+            if (m_inf_heap.size()) {
+                tout << "column " << *m_inf_heap.begin() << " is infeasible" << std::endl;
+                print_column_info(*m_inf_heap.begin(), tout);
+            } else {
+                tout << "x is feasible\n";
+            });
         return m_inf_heap.empty();
     }
     bool current_x_is_infeasible() const { return m_inf_heap.size() != 0; }
-private:
+
+   private:
     lpvar_heap m_inf_heap;
-public:
+
+   public:
     const lpvar_heap& inf_heap() const { return m_inf_heap; }
     lpvar_heap& inf_heap() { return m_inf_heap; }
     void inf_heap_increase_size_by_one() { m_inf_heap.reserve(m_inf_heap.size() + 1); }
     bool inf_heap_contains(unsigned j) const { return m_inf_heap.contains(j); }
-    unsigned inf_heap_size() const { return m_inf_heap.size(); }    
-    indexed_vector<T>     m_pivot_row; // this is the real pivot row of the simplex tableu
-    static_matrix<T, X> & m_A; // the matrix A
+    unsigned inf_heap_size() const { return m_inf_heap.size(); }
+    indexed_vector<T> m_pivot_row;  // this is the real pivot row of the simplex tableu
```

Comment:
it isn't worth applying auto-format in this case.
The alignment introduced manually is destroyed.

Reply:
You can use 

```
// clang-format off
...
// clang-format on
```

To selectively turn clang format off for blocks of code that are manually formatted.